### DISCUSSION TITLE
Does Verdaccio have a configuration option to limit the usage of the current disk space or percentage?

### DISCUSSION THREAD

**Discussion by 89ao** [verdaccio/verdaccio#4987] [Q&A] [answered]
We are encountering a situation where Verdaccio fills up the disk on some nodes, causing the file system to become read-only. Even if we horizontally scale, we still need to perform cleanup. If we could limit the usage, we would only need to scale horizontally.

**Comment by juanpicado** [comment, marked as answer]
No, there is no such thing.

**Comment by 89ao** [comment]
well ,tks @juanpicado
